src/edu/csus/ecs/pc2/clics/API202003/ResourceConfig202003.java

```diff
@@ -0,0 +1,124 @@
+        }
+
+        // CLICS Contest API services are collective -- either all enabled or all disabled (and default to enabled if unspecified)
+        if (wsPropUtilities.getBooleanProperty(WebServerPropertyUtils.CLICS_CONTEST_API_SERVICES_ENABLED_KEY, true)) {
+
+            resConfig.register(new ContestService(getContest(), getController()));
+            showMessage("Starting /contest web service");
+            resConfig.register(new ScoreboardService(getContest(), getController()));
+            showMessage("Starting /contest/scoreboard web service");
+            resConfig.register(new LanguageService(getContest(), getController()));
+            showMessage("Starting /contest/languages web service");
+            resConfig.register(new TeamService(getContest(), getController()));
+            showMessage("Starting /contest/teams web service");
+            resConfig.register(new GroupService(getContest(), getController()));
+            showMessage("Starting /contest/groups web service");
+            resConfig.register(new OrganizationService(getContest(), getController()));
+            showMessage("Starting /contest/organizations web service");
+            resConfig.register(new JudgementTypeService(getContest(), getController()));
+            showMessage("Starting /contest/judgement-types web service");
+            resConfig.register(new ClarificationService(getContest(), getController()));
+            showMessage("Starting /contest/clarifications web service");
+            resConfig.register(new SubmissionService(getContest(), getController()));
+            showMessage("Starting /contest/submissions web service");
+            resConfig.register(new ProblemService(getContest(), getController()));
+            showMessage("Starting /contest/problems web service");
+            resConfig.register(new JudgementService(getContest(), getController()));
+            showMessage("Starting /contest/judgements web service");
+            resConfig.register(new RunService(getContest(), getController()));
+            showMessage("Starting /contest/runs web service");
+            resConfig.register(new EventFeedService(getContest(), getController()));
+            showMessage("Starting /contest/event-feed web service");
+            resConfig.register(new StateService(getContest(), getController()));
+            showMessage("Starting /contest/state web service");
+            resConfig.register(new VersionService(getContest(), getController()));
+            showMessage("Starting / endpoint for version web service");
+
+        }
+
+        return resConfig;
+    }
+
+    // Maybe these should be in a base class that all API's inherit?
+    private void setContestAndController(IInternalContest inContest, IInternalController inController) {
+
+        this.contest = inContest;
+        this.controller = inController;
+        this.log = controller.getLog();
+    }
+
+    private void showMessage(final String message, Exception ex) {
```

Comment:
This method is never called.  I suppose it might be worth leaving it in just for "future support"?   But on the other hand there are no `try/catch` statements in the entire class from which an exception (to be passed to this method) could be caught, so...?

Reply:
Yep. Vestigial code.   I'll remove it.  It's easy enough to re-add if we need it someday.

Reply:
Removed in next push, both here and in the 2023-06 API version of this class (`ResourceConfig202306.java`)